COMP 120 Tinkering Graphics-JH182233/Program/Optionbak

```diff
@@ -0,0 +1,134 @@
+import time
+import sys
+import random
+
+
+#This is that stored pictures
+def season():
+    setMediaPath("E:\COMP 120 Tinkering Graphics\Pictures")
+    global picture1 
+    picture1 = makePicture("Tree-Four-Seasons.jpg")
+    global picture2 
+    picture2 = makePicture("Tree-Winter.jpg")
+    global picture3 
+    picture3 = makePicture("Tree-Spring.jpg")
+    global picture4 
+    picture4 = makePicture("Tree-Summer.jpg")
+    global picture5 
+    picture5 = makePicture("Tree-Autumn.jpg")  
+    global picture6
+    picture6 = makePicture("tree.jpg")
+    global picture7
+    picture7 = makePicture("test.jpg")
+
+
+#The main function for the program
+def main():
+#create an empty picture and populate it with others 
+    season()
+    startX = 0
+    startY = 0
+    global pic
+    pic = makeEmptyPicture(1025, 1110, black) 
+    repaint (pic)
+    time.sleep(1)  
+    copyInto(picture2, pic, startX, startY)
+    repaint (pic)
+    
+    startX = 512
+    startY = 0    
+    time.sleep(1) 
+    copyInto(picture3, pic, startX, startY)
+    repaint (pic)
+    
+    startX = 0
+    startY = 554  
+    time.sleep(1)  
+    copyInto(picture4, pic, startX, startY)
+    repaint (pic)
+    
+    startX = 512
+    startY = 554  
+    time.sleep(1)  
+    copyInto(picture5, pic, startX, startY)
+    repaint (pic)
+    request()
+    
+#Now the program will ask for a season that you would like to view
+def request():
+    startX = 230
+    startY = 300   
+    season = requestString("What season would you like to activate?")
+    if String(season) =="Winter":
+     blackout()
+     copyInto(picture2, pic, startX, startY)
+     repaint(pic)
+    elif String(season) =="Spring":
+     blackout()
+     copyInto(picture3, pic, startX, startY)
+     repaint(pic)
+    elif String(season) =="Summer":
+     blackout()
+     copyInto(picture4, pic, startX, startY)
+     repaint(pic)
+    elif String(season) =="Autumn":
+     blackout()
+     copyInto(picture5, pic, startX, startY)
+     repaint(pic)
+    elif String(season) == "":
+     blackout()
+     fourseasons()
+    time.sleep(3)
+    endSeason = requestString("Would you like another Y/N?")
+    if String(endSeason) == "Y":
+     blackout()
+     show(picture1)
+     return(season)    
```

Comment:
For some reason, why I run the request() or main() function, it ends with "Would you like another Y/N?" but then opens a new window and doesn't use the request() function again, even though it's in the code. Not sure if that's a program error or code issue.